src/platform-integration/web/building.md

```diff
@@ -124,6 +124,12 @@ Run the following command to generate a release build:
 $ flutter build web
 ```
 
+If you receive a `not supported` error, run the following command:
+
+```terminal
+$ flutter config --enable-web
```

Comment:
I'm a little confused on this one. My understanding is that web is enabled by default, and the only time that web will be disabled is when a user has explicitly run `flutter config --no-enable-web` at some point in the past for some reason.

Thoughts @kevmoo ?

Reply:
I'm not familiar w/ this aspect of the CLI *at all*

Reply:
In which case, let's land it as is. For most people this will be a null op. I'd suggest that maybe we think seriously about ripping this functionality out of the CLI, however.